src/qibolab/emulator.py

```diff
@@ -0,0 +1,56 @@
+import pathlib
+
+from qibolab.channels import ChannelMap
+from qibolab.instruments.simulator.pulse_simulator import PulseSimulator
+from qibolab.platform import Platform
+from qibolab.serialize import load_qubits, load_runcard, load_settings
+
+
+def create_runcard_emulator(runcard_folder: str, pulse_simulator: PulseSimulator):
+    """Create a emulator platform using the emulator instrument."""
+    # load runcard
+    runcard_folder_path = pathlib.Path(runcard_folder)
```

Comment:
If you need it, directly ask for a path.

Reply:
Removed in 15307d6